spec/controllers/admin/rooms_controller_spec.rb

```diff
@@ -0,0 +1,171 @@
+require 'spec_helper'
+
+describe Admin::RoomsController do
+  let(:admin) { create(:user) }
+  let(:conference) { create(:conference) }
+  let!(:venue) { create(:venue, conference: conference) }
+  let(:room) { create(:room, venue: venue, size: 4) }
+
+  context 'admin is signed in' do
```

Comment:
Checkout [this discussion](https://github.com/openSUSE/osem/pull/837#discussion_r55192474).